public/stylesheets/overlays/fouquet.css

```diff
@@ -0,0 +1,1192 @@
+html {
+  font-size: 16px;
```

Comment:
Not to disrespect his contribution, but have you thought about alternative names for this besides `fouquet.css`?